### PR TITLE
CONTRIBUTING: update CLA URLs

### PR DESCRIPTION
**Pull request by broady**
/cc @nathany

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ We'd love to accept your patches! Before we can take them, we have to jump a cou
 
 Please fill out either the individual or corporate Contributor License Agreement (CLA).
 
-  * If you are an individual writing original source code and you're sure you own the intellectual property, then you'll need to sign an [individual CLA](http://code.google.com/legal/individual-cla-v1.0.html).
-  * If you work for a company that wants to allow you to contribute your work, then you'll need to sign a [corporate CLA](http://code.google.com/legal/corporate-cla-v1.0.html).
+  * If you are an individual writing original source code and you're sure you own the intellectual property, then you'll need to sign an [individual CLA](https://developers.google.com/open-source/cla/individual).
+  * If you work for a company that wants to allow you to contribute your work, then you'll need to sign a [corporate CLA](https://developers.google.com/open-source/cla/corporate).
```
Comment (nathany) [inline CONTRIBUTING.md]: https://cla.developers.google.com/about/google-corporate ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ We'd love to accept your patches! Before we can take them, we have to jump a cou
 
 Please fill out either the individual or corporate Contributor License Agreement (CLA).
 
-  * If you are an individual writing original source code and you're sure you own the intellectual property, then you'll need to sign an [individual CLA](http://code.google.com/legal/individual-cla-v1.0.html).
-  * If you work for a company that wants to allow you to contribute your work, then you'll need to sign a [corporate CLA](http://code.google.com/legal/corporate-cla-v1.0.html).
+  * If you are an individual writing original source code and you're sure you own the intellectual property, then you'll need to sign an [individual CLA](https://developers.google.com/open-source/cla/individual).
```
Comment (nathany) [inline CONTRIBUTING.md]: https://cla.developers.google.com/about/google-individual ?

Comment (broady) [inline CONTRIBUTING.md]: Indeed, it looks like the URLs had changed again. I copied these from @GoogleCloudPlatform/golang-samples :(

Comment (nathany) [inline CONTRIBUTING.md]: 😄 

"the only thing that's constant is change"

**Pull request merged**